pilot/pkg/serviceregistry/kube/controller/controller.go

```diff
@@ -992,7 +992,7 @@ func (c *Controller) getProxyServiceInstancesFromMetadata(proxy *model.Proxy) ([
 
 	}
 	if len(services) == 0 {
-		return nil, fmt.Errorf("no instances found for %s: %v", proxy.ID, err)
```

Comment:
why remove this warning? We don't support proxy without service

Reply:
ok, will remove